doc/ReconstructingMeasurements.rst

```diff
@@ -84,3 +84,58 @@ id for each object.
     # Re-run measure on the sources selected above, using the reconstructed
     # noise replacer.
     measTask.runPlugins(noiseReplacer, newSrcCatalog, exposure)
+
+
+At the moment, measTask.runPlugins will not run on any child
+objects if their parents are not also in the catalog. This issue can be
+solved in several ways, two of which are options in makeRerunCatalog. The
+``resetParents`` flag (True by default) will reset the parent keys of any
+children whose parents are not included in the idList, effectively turning
+them into parents. The ``addParents`` flag (False by default) will add these
+parents to the idList. Some timed examples follow:
+
```

Comment:
In the example below, all the timing code is unrelated to the topic of this document, and obfuscates the point being made. Examples should have the bare minimum amount of extra utility and concepts to demonstrate a point. 

Consider instead counting the number of nans (or zeros if that is the default) in a column (pick which ever makes sense)  before and after turning on resetParents, to show that the number decreases. You can also show that before turning on reset parents the number of bad values is equal to the number of sources that are children. To demonstrate adding parents, you can show the number of bad values as above, but you can also show the length of the inputId list vs the length of the output rerun catalog and how the later is now larger.

Reply:
:+1: